io/prometheus/client/metrics.proto

```diff
@@ -46,6 +46,7 @@ message Gauge {
 message Counter {
   optional double   value    = 1;
   optional Exemplar exemplar = 2;
+  optional int64    created  = 3;
```

Comment:
Nice! I would vote for unit in the name so we protocol users can clearly pass that OR the timestamp type.

Also "time" or "ts" or "timestamp" word in the name would help too.

E.g. following what Exemplar message has:

```suggestion
optional google.protobuf.Timestamp created_timestamp = 3;
```

The alternative:

```suggestion
  optional int64 created_timestamp_ms = 3;
```

What would be your choice @beorn7 ?

Reply:
Sounds like `google.protobuf.Timestamp` might be a better choice: https://github.com/prometheus/client_model/pull/39#issuecomment-575535354